fractions/microprofile/microprofile-metrics/README.adoc

```diff
@@ -12,7 +12,7 @@ To use this in your project you need the following in your pom.xml
     </dependency>
 ----
 
-There is no need to include the MicroProfile Metrics API dependency, as it comes with the fraction.
+There is no need to include the  MicroProfile Metrics API dependency, as it comes with the fraction.
```

Comment:
```suggestion
There is no need to include the MicroProfile Metrics API dependency, as it comes with the fraction.
```

Reply:
Thanks for catching the extra space. My bad:)